.github/workflows/ci.yml

```diff
@@ -9,13 +9,23 @@ jobs:
           - "3.1"
           - "3.2"
           - "3.3"
+          - "3.4"
         gemfile:
           - gemfiles/rails_6_1.gemfile
-          - gemfiles/rails_7_0.gemfile
```

Comment:
Are we intentionally dropping `7.0` from the gemfiles list? The file is still there so I assumed not

Reply:
nope that was a mistake, fixed.